docs/recipes/custom-gatherer-puppeteer/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
+
+yarn
+node lighthouse --config-path=custom-config.js https://www.example.com --output=json \
+  | jq '.audits["searchable-audit"].score' \
```

Comment:
I think `jq` is introducing a new dependency for running tests. can we document that somewhere, maybe near the protobuf installation bit?

Reply:
This test won't run for `yarn test`.

Reply:
still :)

Reply:
I don't know how to word it next to the protobuf bit - that section is for how to contribute, and only mentions protobuf "if you are going to make changes that affect the output". `jq`, for this test, only runs in CI, and what will break it is unknown, and to run it manually you gotta do `yarn test-docs`. That seems like a lot of irrelevant info for something that should be rare, and fixed immediately by one of us.

any ideas?

Reply:
what about?

```md
## Docs

Some of our docs have tests that run only in CI by default. If you end up needing to modify our documentation, you'll need to run `yarn test-docs` locally to make sure they pass.

**Additional Dependencies**
`brew install jq`

Reply:
thanks!